herokuappTests/base/base_class.py

```diff
@@ -0,0 +1,7 @@
+from  auth.authenticator import Authenticator
+from  utility.booking_api_utils import BookingApiUtils
+
+class Base:
+    def __init__(self):
```

Comment:
I think you cannot have constructor here since it is base test directly inherited from pytest class. 
I was unable to run this example like this.

However, you can do two things:
```
class Base:
    token = Authenticator().get_token()
    booking_api = BookingApiUtils(token)
```

or:
```
class Base:
    def setup_method(self):
        self.token = Authenticator().get_token()
        self.booking_api = BookingApiUtils(self.token)
```

One will behave like "beforeAll", another like "beforeEach"